website/assets/scss/_variables_project.scss

```diff
@@ -3,3 +3,75 @@
 Add styles or override variables from the theme here.
 
 */
+
+// Color palatte (AWS branding)
```

Comment:
This doesn't feel super neutral to me.

Reply:
Had to start somewhere, to be fair this is no longer entirely accurate in that we primarily use non-AWS colors. I can update this to better reflect that

Reply:
Updated to include only necessary colors